test/actions/websocket.test.js

```diff
@@ -595,4 +595,49 @@ describe('Actions.Websocket', () => {
 
         test();
     });
+
+    it('handle license changed', (done) => {
+        async function test() {
+            if (TestHelper.isLiveServer()) {
+                // No live server version implemented for this test case.
+                this.skip();
```

Comment:
Running `yarn test-no-mock` is broken for all sorts of tests right now, so I figured I wouldn't spend a lot of time on this. Pinged the developers channel for discussion on this point.